ui/appui-react/src/test/layout/widget-panels/Content.test.snap

```diff
@@ -0,0 +1,31 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`WidgetPanelsContent should render 1`] = `
```

Comment:
Avoid adding new snapshot tests. We are trying to get rid of these in all our components, use react testing library queries instead.